src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -201,6 +214,26 @@ private static String getUserId(String userName) {
     }
   }
 
+  private static HttpHost getBorderProxy() {
+
+    EnvType envType = Configuration.getEnvType(Configuration.getEnv());
+    File configFile = new File(Configuration.getCredentialsFilePath());
+
+    switch (envType) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1301)